web/styles/compose.css

```diff
@@ -661,21 +662,19 @@ input.recipient_box {
     }
 
     .compose_control_button {
-        padding: 5px;
+        padding: 0.7px 5px 0;
         opacity: 0.7;
         color: inherit;
         text-decoration: none;
         font-size: 17px;
-        text-align: center;
 
         &:hover {
             opacity: 1;
         }
     }
 
-    .fa-eye {
-        position: relative;
-        top: -0.7px;
+    .markdown_preview {
+        padding-top: 0;
```

Comment:
Is there someplace else in the CSS or via JS that `.markdown_preview` gets a padding-top value? I'm curious about the zero value and whether this could be removed entirely.